packages/kernel-shims/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0]
+
+### Changed
+
+- Bump endo dependencies ([#543](https://github.com/MetaMask/ocap-kernel/pull/543))
```

Comment:
We don't need to release this package. This was the only change, and the bumps were all minor, so consumers can still dedupe.